client/extensions/woocommerce/state/ui/shipping/zones/test/selectors.js

```diff
@@ -41,12 +41,42 @@ describe( 'selectors', () => {
 			const state = {
```

Comment:
I think helpers should make easier to set up the common case, and here the common case by far is to have reducers/selectors operating on a single site. We don't even have multi-site UI yet.

If that's the only problem you have with this solution, I think we can live with that :)